api/Window.json

```
@@ -883,10 +883,10 @@
               "version_added": true
             },
             "ie": {
-              "version_added": true
+              "version_added": "4"
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
```

```suggestion
              "version_added": true
```

Oops, this isn't IE/EdgeHTML, I guess this is just `git add -p` gone wrong :)